src/fusor/models.py

```diff
@@ -216,32 +202,17 @@ class LinkerElement(BaseStructuralElement, extra="forbid"):
     type: Literal[FUSORTypes.LINKER_SEQUENCE_ELEMENT] = (
         FUSORTypes.LINKER_SEQUENCE_ELEMENT
     )
-    linker_sequence: SequenceDescriptor
-
-    @field_validator("linker_sequence", mode="before")
-    def validate_sequence(cls, v):
-        """Enforce nucleotide base code requirements on sequence literals."""
-        if isinstance(v, dict):
-            try:
-                v["sequence"] = v["sequence"].upper()
-            except KeyError as e:
-                raise TypeError from e
-        elif isinstance(v, SequenceDescriptor):
-            v.sequence = v.sequence.upper()
-        else:
-            raise TypeError
-
-        return v
+    linkerSequence: LiteralSequenceExpression
 
     model_config = ConfigDict(
+        arbitrary_types_allowed=True,
```

Comment:
I'm surprised we have to set this here, what's causing it?

Reply:
oops, so I'm not sure why this is appearing in the diff. I have no recollection of changing this